client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```diff
@@ -476,3 +476,44 @@
 .ReactModal__Html--open.lightbox-mode .layout__content {
 	overflow-y: scroll;
 }
+
+.product-lightbox__included-product-list-label {
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.product-lightbox__included-product-list-item {
+	display: flex;
+	gap: 12.57px;
```

Comment:
Though I don't think using decimal values for pixels can still create cross-browsers inconsistencies, I believe the consensus is to stick to integers. It makes it easier to read and to compute other values.

I think there's also a will to pick spacing values from a scale rather than to use arbitrary numbers.

Reply:
I totally agree on this. nice suggestion @monsieur-z  👍